### PR TITLE
Fixed issue with .js extension in the imports

### PR DESCRIPTION
**Pull request by shiva-beehyv**
Facing issue with importing the sdk. Unable to resolve the modules in the app and it is expecting .js extension in the imports inside the sdk

### CODE REVIEW

File: lib/services/verification.service.js [hunk 1]
```diff
@@ -6,10 +6,9 @@ import {Ed25519Signature2018} from "@digitalbazaar/ed25519-signature-2018";
 import {LDKeyPair as RSAKeyPair} from "crypto-ld";
 import {Ed25519Signature2020} from "@digitalbazaar/ed25519-signature-2020";
 import * as vc from "@digitalbazaar/vc";
-import { createRequire } from 'module';
```
Comment (shiva-beehyv) [inline lib/services/verification.service.js:9]: module is not available in browser. so, updated the import part from jsonld-signatures

File: package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "@sunbird-rc/verification-sdk",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "An SDK to verify Sunbird generated Verifiable Credentials",
   "main": "./lib/index.js",
   "type": "module",
   "scripts": {
-    "test": "mocha --reporter spec test/*.spec.js"
+    "test": "mocha --require @babel/register --reporter spec test/*.spec.js "
```
Comment (shiva-beehyv) [inline package.json:8]: tests were failing after adding the .js extension. so added babel configuration to run the tests

File: lib/services/verification.service.js [hunk 3]
```diff
@@ -23,11 +22,15 @@ const getDocumentLoader = (didDoc) => {
                 contextUrl: null, documentUrl: url, document: DOCUMENTS[url]
             }
         }
-        return await jsonld.documentLoaders.node()(url);
-    });
+        let document = await (await fetch(url)).json();
```
Comment (shiva-beehyv) [inline lib/services/verification.service.js:25]: Facing following error with jsonld node document loader:
jsonld.InvalidUrl: Dereferencing a URL did not result in a valid JSON-LD object. Possible causes are an inaccessible URL perhaps due to a same-origin policy (ensure the server uses CORS if you are using client-side JavaScript), too many redirects, a non-JSON response, or more than one HTTP Link Header was provided for a remote context.

So, updated the part with fetch

Comment (holashchand) [reply lib/services/verification.service.js:25]: Did you try without this change and with all other changes?

**Review by holashchand — APPROVED** [review]

**Review by challabeehyv — APPROVED** [review]

**Comment by shiva-beehyv** [comment]
Fix covered as part of [another pr](https://github.com/Sunbird-RC/verification-sdk/pull/4)

**Pull request closed without merging**